threestudio/systems/gaussian_splatting.py

```diff
@@ -0,0 +1,256 @@
+import os
+import math
+from dataclasses import dataclass
+
+import torch
+import numpy as np
+
+import threestudio
+from threestudio.systems.base import BaseLift3DSystem
+from threestudio.models.geometry.gaussian import BasicPointCloud
+from threestudio.utils.typing import *
+
+def getWorld2View2(R, t, translate=np.array([.0, .0, .0]), scale=1.0):
+    Rt = np.zeros((4, 4))
+    Rt[:3, :3] = R.transpose()
+    Rt[:3, 3] = t
+    Rt[3, 3] = 1.0
+
+    C2W = np.linalg.inv(Rt)
+    cam_center = C2W[:3, 3]
+    cam_center = (cam_center + translate) * scale
+    C2W[:3, 3] = cam_center
+    Rt = np.linalg.inv(C2W)
+    return np.float32(Rt)
+
+def getProjectionMatrix(znear, zfar, fovX, fovY):
```

Comment:
I noticed something interesting – this function seems a bit different from the one in ```utils/ops.py``` called ```get_projection_matrix```. Could you help me understand the differences?